### ISSUE TITLE
GoNeovim crashes when connecting two clients to one server

### ISSUE THREAD

**Issue by glacambre** [akiyosi/goneovim#87]
OS Version: Linux 5.0.0-37-generic 40~18.04.1-Ubuntu SMP Thu Nov 14 12:06:39 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
Neovim version: NVIM v0.5.0-279-ge922576bd
GoNeovim version: 0.4.2

GoNeovim crashes when trying to connect two GoNeovim clients to a single neovim server. Steps to reproduce:
- Start GoNeovim
- Get the server address with `:echo $NVIM_LISTEN_ADDRESS`
- Connect to that address with `goneovim --server=…`

This results in the following error message:
```
msgpack/rpc: session closed
read unix @->/tmp/nvimmaxcCN/0: read: connection reset by peer
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x971276]

goroutine 14 [running]:
github.com/neovim/go-client/nvim.(*Nvim).call(0x0, 0xd07231, 0xc, 0x0, 0x0, 0xc00027a480, 0x1, 0x1, 0xc0000163a8, 0xe1ada0)
        github.com/neovim/go-client/nvim/nvim.go:406 +0x26
github.com/neovim/go-client/nvim.(*Nvim).Command(0x0, 0xcff6d6, 0x3, 0xc000233690, 0x830e96)
        github.com/neovim/go-client/nvim/apiimp.go:740 +0xb6
created by github.com/akiyosi/goneovim/editor.(*MiniMap).exit
        github.com/akiyosi/goneovim/editor/minimap.go:157 +0x5b
```

**Comment by akiyosi** [comment]
Hi :D
Thanks for this issue report. Thanks also for the informative discussion in the gitter.

I made a fix at commit number b00298249e15198151e0916a85453652ea5caaf8 to improve this problem.
This problem will be improved in the next release.

**Comment by akiyosi** [comment]
This problem has been fixed in v0.4.4
Is it possible to confirm the problem?

**Comment by glacambre** [comment]
Yup, I can confirm that this is fixed with Goneovim-0.4.4 :)
